CHANGELOG/CHANGELOG-1.3.md

```
@@ -0,0 +1,964 @@
+## Provider-specific Notes
+
+* AWS
+  * Support for ap-northeast-2 region (Seoul)
+  * Allow cross-region image pulling with ECR
+  * More reliable kube-up/kube-down
+  * Enable ICMP Type 3 Code 4 for ELBs
+  * ARP caching fix
+  * Use /dev/xvdXX names
+  * ELB:
+    * ELB proxy protocol support 
+	* mixed plaintext/encrypted ports support in ELBs
```

markdownlint-cli2 (0.22.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 4; Actual: 1

(MD005, list-indent)